mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -161,18 +169,8 @@ LogicalResult PopulateParams::calculateBlockGemmPerformanceParameters(
   return success();
 }
 
-LogicalResult PopulateParams::populateDerived(const InitParamsNonAccel &params,
-                                              GemmSize &gemmSize,
-                                              uint32_t &gridSize) {
-  auto gemmExtraPad =
-      calculatePadding(params.gemmKPerBlock, params.gemmMPerBlock,
-                       params.gemmNPerBlock, gemmSize);
-  if (gemmExtraPad.has_value()) {
-    gemmSize.m += gemmExtraPad->m;
-    gemmSize.k += gemmExtraPad->k;
-    gemmSize.n += gemmExtraPad->n;
-  }
-
+LogicalResult
+PopulateParams::populateDerived(const InitParamsNonAccel &params) {
```

Comment:
why do we need this function now ? It feels like redundant wrapper right ?

Reply:
feel free to ignore on the basis... non_accel path has a short life time.

Reply:
1. Validation
2. We might end up getting more derived parameters in the future, let's keep the structure in there.